### ISSUE TITLE
Find in folder not working

### ISSUE THREAD

**Issue by Couim** [microsoft/vscode#103852]
Issue Type: <b>Bug</b>

Hi, 
I have multiple .log files and I want to search every occurencies of "build_docker" pattern. I know that in this folder, there is around 100 occurencies in several files. When I do the following steps, it display 0 results : 

1. Open vs code
2. Open a folder
3. Go to Edit > Find in files (ctrl + shift + F)
4. Search for something you know it is inside your folder but it will not find. 

Note : it works if I open every files in the "Open Editor" section in the Explorer window.

VS Code version: Code 1.47.3 (91899dcef7b8110878ea59626991a18c8a6a1b3e, 2020-07-23T13:12:49.994Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz (4 x 2496)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.78GB (1.30GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.36.2
python|ms-|2020.7.96456

</details>

**Comment by IllusionMH** [comment]
/needsMoreInfo

Are you sure that this folder or these files aren't ignored in `.gitignore`?

If you have **Use Exclude Settings and Ignore Files** enabled as on screenshot - VS Code will avoid searching in files/folders in `.gitignore` file, `files.exclude` and `search.exclude` settings
![image](https://user-images.githubusercontent.com/1312662/89219304-2cd48980-d5d8-11ea-814d-7c5f678e6dd4.png)

_(unless you have editor open, which will show results and it's a bug)_

**Comment by Couim** [comment]
Hi @IllusionMH

Yes normally I've nothing inside my working folder. No .gitignore files, it's a folder populated only by .log files & txt files.
I tested to find with the "Use Exclude Settings and Ignore Files" enabled and disabled and it's the same result. 

![image](https://user-images.githubusercontent.com/8838231/89224096-2fcf7a00-d5d8-11ea-9fdb-696ec9ba3bbe.png)
![image](https://user-images.githubusercontent.com/8838231/89224189-57264700-d5d8-11ea-814d-55df576770ea.png)

If I open all my files in Open Editor section I get this : 
![image](https://user-images.githubusercontent.com/8838231/89224322-9359a780-d5d8-11ea-9e72-f01c14a91e09.png)

**Comment by AllSeeingEye** [comment]
I can confirm that this is broken in recent build. Only open files are being searched.

**Comment by IllusionMH** [comment]
Could you please check whether it works in Insiders build? https://code.visualstudio.com/insiders/

**Comment by Couim** [comment]
Hi @IllusionMH 
I had the same problem with insiders build. But it looks like the problem comes from my files .log. 
I tried in another folder with txt files and the search function works as expected except with my .log files. Maybe it's a problem of format ? I generated them with Mobaxterm logging feature. I can send a sample file if someone wants to reproduce the problem and work on it ?

**Comment by GregoryNatter** [comment]
I tested to reproduce your issue. I observed that the "Find in folder" doesn't check on file with spaces on their names.

**Comment by roblourens** [comment]
@Couim maybe an encoding issue? Are those files not utf-8, can you set `files.encoding`?

**Comment by Couim** [comment]
Hello @roblourens, thanks for your answer. I think my files are corrupted. 

VScode says it's UTF-8 but I think I got problems on encoding. I tried also in CRLF and LF but it still the same. 
If I want to reencode it un UTF16-LE for example, I get this message from VSCode.
![image](https://user-images.githubusercontent.com/8838231/89505778-6ff64e80-d7ca-11ea-9274-ad1646430af8.png)

If I click on "Fix this file" I get asian characters & special character mix in my files : 
![image](https://user-images.githubusercontent.com/8838231/89505923-9ddb9300-d7ca-11ea-9a08-971fd2efddfe.png)

Maybe the issue come from MobaXterm encoding system ? My files come from this software.
